server.js

```diff
@@ -15,6 +15,7 @@ app.route('/').get((req, res) => {
   res.render('Load your view here');
 });
 
-app.listen(process.env.PORT || 3000, () => {
-  console.log('Listening on port ' + process.env.PORT);
-});
+const PORT = process.env.PORT || 3000;
+app.listen(PORT, () => {
+  console.log('Listening on port ' + PORT);
+});
```

Comment:
```suggestion
});

```
Could we add the EOF new line?

Otherwise, this looks good to me.

Reply:
Great. Let me work on it.